### PR TITLE
Bump 4.0.0.pre7

### PR DESCRIPTION
**Pull request by jasonrclark**
# Release plan

* [x] Rebase this branch once #1316 is merged (which touched version files)
* [x] Retest all the samples
* [x] Build preliminary gems, install and test on:
  * [x] Mac
  * [x] Windows
  * [x] Linux (Ubuntu)
* [x] 🚢 
* [ ] Give a shot at seriously fixing packaging for [`pre8`](https://github.com/shoes/shoes4/milestone/11) 😨

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,33 @@
...
+-------------------------------
+
+Improvements (5)
+----------------
+
+* Implement translate method on application [1280e92]
+* Implement arrow art element [b9cd00b]
+* Allow attaching elements via style [9896ef2]
+* Remove adjusted versions of samples [9e1ea56, 762a395, a2a9a88, ff0adff]
+* Numerous Rubocop rules now in force! Now Ruby 2.3 flavored!
```
Comment (PragTob) [inline CHANGELOG]: Imo that's nothing for the changelog (and the one above is also questionable) as it's not really user facing so far as far as I can tell :)

Comment (jasonrclark) [inline CHANGELOG]: Totally cool. Here comes the ✂️ !

**Comment by jasonrclark** [comment]
Testing on all three OS's went well. Will cut the release this weekend when I get another slice of time! 💃

**Comment by PragTob** [comment]
Thank you so much for your diligent work @jasonrclark :heart:

**Pull request merged**